packages/dds/tree/src/shared-tree/schematizingTreeView.ts

```diff
@@ -266,14 +258,9 @@ export function requireSchema<TRoot extends FlexFieldSchema>(
 		viewSchema.schema,
 		nodeKeyManager,
 		nodeKeyFieldKey,
-		() => {
-			const deleted = slots.delete(ViewSlot);
-			assert(deleted, 0x8c4 /* unexpected dispose */);
-			onDispose();
-		},
+		onDispose,
```

Comment:
Why don't we check for the extra dispose anymore?

Reply:
That logic moved to Context.dispose (which runs as part of view disposal)